account-abstraction/schemas.yaml

```diff
@@ -6,54 +6,37 @@ components:
         maxSpendUsd:
           type: string
           description: Maximum amount of USD that can be spent by the Gas Manager
-          default: '5000.00'
-          nullable: true
```

Comment:
nullable should still be true right? All rules and solanaRules fields are nullable

Reply:
I knew you'd catch that haha -- should've been more descriptive.

I realized there was no need for `nullable` as by default they are nullable!

![image](https://github.com/user-attachments/assets/0c3d2420-7914-4d65-a364-a9b476727163)